File: JavaScriptBasics/AdvancedConditionalStatements/Exercise/NewHome.js
function NewHome(input){

let flower = input[0];
let count = Number(input[1]);
let budget = Number(input[2]);
let total;

switch(flower){
    case "Roses" :
       total = count*5;
       if(count>80){ total=total*0.9;}break;
    case "Dahlias" :
        total = count*3.80;
        if(count>90){
            total=total*0.85;}break;
    case "Tulips" : 
              total = count*2.80;
        if(count>80){
            total=total*0.85;
        }break;
    case "Narcissus" :
        total = count*3;
        if(count<120){
            total=total*1.15;
        }break;
    case "Gladiolus" :
        total = count*2.50;
        if(count<80){
            total=total*1.20;
        }break;
}

if(budget>total){
    console.log("Hey, you have a great garden with " + count+" "+ flower+" and "+(budget-total).toFixed(2) +" leva left.");
}else{
    console.log("Not enough money, you need " +(total-budget).toFixed(2) +" leva more.");
}

}

File: JavaBasics/AdvancedConditionalStatements/Exercise/NewHome.java
package EntryModule.AdvancedConditionalStatements.Exercise;

import java.util.Scanner;
public class NewHome {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String flowersname = scanner.nextLine();
        int number = Integer.parseInt(scanner.nextLine());
        int budget = Integer.parseInt(scanner.nextLine());
        double price;

        if (number > 0) {
            switch (flowersname) {
                case "Roses":
                    if (number > 80) {
                        price = 4.5;
                    } else {
                        price = 5.00;
                    }
                    if (Math.abs(price * number) <= budget) {
                        System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", number, flowersname, budget - (price * number));
                    } else {
                        System.out.printf("Not enough money, you need %.2f leva more.", (price * number) - budget);
                    }
                    break;
                case "Dahlias":
                    if (number > 90) {
                        price = 3.23;
                    } else {
                        price = 3.80;
                    }
                    if (Math.abs(price * number) <= budget) {
                        System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", number, flowersname, budget - (price * number));
                    } else {
                        System.out.printf("Not enough money, you need %.2f leva more.", (price * number) - budget);
                    }
                    break;
                case "Tulips":
                    if (number > 80) {
                        price = 2.38;
                    } else {
                        price = 2.80;
                    }
                    if (Math.abs(price * number) <= budget) {
                        System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", number, flowersname, budget - (price * number));
                    } else {
                        System.out.printf("Not enough money, you need %.2f leva more.", (price * number) - budget);
                    }
                    break;
                case "Narcissus":
                    if (number < 120) {
                        price = 3.45;
                    } else {
                        price = 3.00;
                    }
                    if (Math.abs(price * number) <= budget) {
                        System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", number, flowersname, budget - (price * number));
                    } else {
                        System.out.printf("Not enough money, you need %.2f leva more.", (price * number) - budget);
                    }
                    break;
                case "Gladiolus":
                    if (number < 80) {
                        price = 3.00;
                    } else {
                        price = 2.50;
                    }
                    if (Math.abs(price * number) <= budget) {
                        System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", number, flowersname, budget - (price * number));
                    } else {
                        System.out.printf("Not enough money, you need %.2f leva more.", (price * number) - budget);
                    }
                    break;

            }
        }
    }}
